artifacts/openshift/vsphere-kubernetes-drivers-operator.clusterserviceversion.yaml

```diff
@@ -106,11 +107,8 @@ spec:
           - description: 'Informs the status of vSphere Storage Provider'
             displayName: 'Storage Provider'
             path: csi
-  description: 'Vsphere Kubernetes Drivers Operator helps to install/manage vsphere drivers required to run kubernetes workload on vSphere. The operator currently manages CPI and CSI vSphere drivers. The operator exposes two resources: VsphereCloudConfig - represents the connection information required to connect to vcenter. VDOConfig - represents the configuration information w.r.t CPI and CSI'
-  displayName: Vsphere Kubernetes Drivers Operator
-  icon:
```

Comment:
do we not require this anymore

Reply:
Icon length is too big, and the yaml  lint is failing this part, we need to find a way to put this in a different way, IMO as of now we can skip this